docs/getting-started-guides/index.md

```diff
@@ -46,6 +46,7 @@ clusters.
 [AppsCode.com](https://appscode.com/products/cloud-deployment/) provides managed Kubernetes clusters for various public clouds (including AWS and Google Cloud Platform).
 
 [KCluster.io](https://kcluster.io) provides highly available and scalable managed Kubernetes clusters for AWS.
+[KUBE2GO.io] (https://kube2go.io) get started with highly available Kubernetes clusters on multiple public clouds along with useful tools for development, debugging, monitoring.
```

Comment:
Formatting: Add an extra newline after line 48, and remove the space between ']' and '(' to fix the link. Currently, it looks like this:
![screen shot 2017-02-23 at 10 34 35 am](https://cloud.githubusercontent.com/assets/24798125/23273410/be305d0a-f9b3-11e6-9aac-fc9656f82899.png)